planning/behavior_path_avoidance_module/src/utils.cpp

```diff
@@ -418,33 +418,41 @@ bool isParkedVehicle(
 
   bool is_left_side_parked_vehicle = false;
   if (!isOnRight(object)) {
-    auto [object_shiftable_distance, sub_type] = [&]() {
-      const auto most_left_road_lanelet =
-        route_handler->getMostLeftLanelet(object.overhang_lanelet);
-      const auto most_left_lanelet_candidates =
-        route_handler->getLaneletMapPtr()->laneletLayer.findUsages(
-          most_left_road_lanelet.leftBound());
-
-      lanelet::ConstLanelet most_left_lanelet = most_left_road_lanelet;
+    const auto most_left_lanelet = [&]() {
+      const auto same_direction_lane =
+        route_handler->getMostLeftLanelet(object.overhang_lanelet, true, true);
       const lanelet::Attribute sub_type =
```

Comment:
up to Ota-san's judgement but here can be `const lanelet::Attribute & sub_type` based on `clang-tidy` result.

Reply:
Fixed in https://github.com/autowarefoundation/autoware.universe/pull/6731/commits/4e28acd86a6d8e4cadf6cbdcc399d8cd5799887d.